openstack/tempest/tempest-base/templates/_tempest-pod.yaml.tpl

```diff
@@ -6,6 +6,10 @@ metadata:
   labels:
     system: openstack
     type: configuration
+  annotations:
```

Comment:
Let's skip the linkerd annotation for now

I think we can treat tempest as an external client, not part of the internal mesh